defrag/defrag.go

```diff
@@ -0,0 +1,293 @@
+package ip4defrag
```

Comment:
Would you mind adding a package comment here?

Also, move file into dir ip4defrag/ (instead of defrag/) if you're going to use that package name.